src/core/diags/factory_diags.cpp

```diff
@@ -758,7 +758,11 @@ void Diags::AlarmFired(void)
     {
         uint32_t now = otPlatAlarmMilliGetNow();
 
-        TransmitPacket();
+        Error error = TransmitPacket();
+        if (error != kErrorNone)
+        {
+            Output("Packet transmission failed\r\nstatus %#x\r\n", error);
```

Comment:
Because continuous sending errors may occur, it is more convenient for users to see the error information one by one. Could we change the message like this.

```c++
Output("Packet transmission failed, error: %s\r\n", ErrorToString(error));
```